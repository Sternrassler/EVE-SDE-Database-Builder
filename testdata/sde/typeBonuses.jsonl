{"typeID":587,"bonusID":1,"bonusValue":5.0,"importance":1,"unitID":null}
{"typeID":588,"bonusID":2,"bonusValue":10.0,"importance":2,"unitID":1}
{"typeID":606,"bonusID":3,"bonusValue":7.5,"importance":1,"unitID":null}
