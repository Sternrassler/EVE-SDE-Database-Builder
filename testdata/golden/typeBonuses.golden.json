[
  {
    "typeID": 587,
    "bonusID": 1,
    "bonusValue": 5,
    "bonusText": null,
    "importance": 1,
    "unitID": null
  },
  {
    "typeID": 588,
    "bonusID": 2,
    "bonusValue": 10,
    "bonusText": null,
    "importance": 2,
    "unitID": 1
  },
  {
    "typeID": 606,
    "bonusID": 3,
    "bonusValue": 7.5,
    "bonusText": null,
    "importance": 1,
    "unitID": null
  }
]